src/components/Loading/README.md

```diff
@@ -67,19 +67,36 @@ The loading component should:
 
 ---
 
-## Content guidelines
+## Related components
```

Comment:
I took a look through the style guide and it seems we haven't got a consistent way of documenting "Related components". 

Sometimes component names are:
- capitalized
- preceded with "the"
- followed by "component"

My preference is to use lowercase, "the", and "component": "the spinner component". BUT that might be out of scope here, especially since the other components in this section look like the content that's been added here. I think as long as the pattern is consistent, then for now that's 👌

Reply:
@sadiesaurus Would it make sense to fold that into the content audit that (I think?) is happening? I think we could do a bunch of those formatting updates as part of separate PRs.

Reply:
@dpersing It _is_ happening 😄 and that's a great idea. I will add this in.